hw/ip/rstmgr/doc/_index.md

```diff
@@ -284,31 +284,33 @@ Reset Cause             | Description
 `POR`                   | Cold boot, the system was reset through POR circuitry.
 `LOW_POWER_EXIT`        | Warm boot, the system was reset through low power exit.
 `NDM RESET`             | Warm boot, the system was reset through `rv_dm` non-debug-module request.
+`SW_REQ`                | Warm boot, the system was reset through {{< regref "RST_REQ" >}}.
 `HW_REQ`                | Warm boot, the system was reset through peripheral requests.  There may be multiple such requests.
 
 
 The reset info register is write 1 clear.
 It is software responsibility to clear old reset reasons; the reset manager simply records based on the rules below.
 
 Excluding power on reset, which is always recorded when the device POR circuitry is triggered, the other resets are recorded when authorized by the reset manager.
-Reset manager authorization in turn is based on reset category as indicated by the power manager.
-The power manager observes 3 categories of reset states that are mutually exclusive.
+Reset manager authorization in turn is based on reset categories as indicated by the power manager.
+The power manager observes 4 categories of reset states that are mutually exclusive.
 *   No resets are triggered through the power manager.
```

Comment:
I don't see why "No " here. I assume this refers to resets that are triggered by pwrmgr, like power glitch or escalation.

Reply:
i hvae no idea what i was trying to say there. Let me reword.

Reply:
o i remember now, i was trying to say one of the cases is that no resets are triggered through the pwrmgr (ndm_rst)